game/scripts/npc/abilities/crystal_maiden_frostbite.txt

```diff
@@ -27,11 +27,11 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "9 8 7 6 6 6"
+    "AbilityCooldown"                                     "6" //OAA
```

Comment:
As far as I know this is the part of Crystal Maiden that doesn't need buffing, especially because she can perma-root people with lvl 25 talent. But whatever.